pkg/operator/staticpod/controllers.go

```diff
@@ -281,26 +278,28 @@ func (b *staticPodOperatorControllerBuilder) ToControllers() (manager.Controller
 		eventRecorder.Warning("PruningControllerMissing", "not enough information provided, not all functionality is present")
 	}
 
-	manager.WithController(startupmonitorcondition.New(
-		b.operandNamespace,
-		b.staticPodName,
-		b.staticPodOperatorClient,
-		b.kubeInformers,
-		b.enableStartMonitor,
-		eventRecorder,
-	), 1)
+	if b.enableStartMonitor != nil {
+		manager.WithController(startupmonitorcondition.New(
+			b.operandNamespace,
+			b.staticPodName,
+			b.staticPodOperatorClient,
+			b.kubeInformers,
+			b.enableStartMonitor,
+			eventRecorder,
+		), 1)
 
-	if b.operandPodLabelSelector.Empty() {
-		errs = append(errs, fmt.Errorf("missing OperandPodLabelSelector when running StaticPodFallbackConditionController; cannot proceed"))
-	} else {
-		manager.WithController(staticpodfallback.New(
+		if staticPodFallbackController, err := staticpodfallback.New(
```

Comment:
Having this controller under the `enableStartMonitor`gate feels a bit odd since all the other controllers have their own parameters in the builder that they can be gated on. I can see why we could want it under `enableStartMonitor` since it goes along with the monitoring, but for consistency purposes, it might be better to have it separated from the startup monitor controller.

Reply:
Had an offline discussion with @p0lyn0mial who told me that these controllers were under the same gate because they were part of the same story.